flow/project.py

```diff
@@ -3786,24 +4114,43 @@ def operation_function(job):
         except KeyError:
             raise KeyError("Unknown operation '{}'.".format(args.operation))
 
-        for job in jobs:
-            operation_function(job)
+        for job in self.aggregates[args.operation]:
```

Comment:
I also thought of this once.
We could do this is #336 marking all those functions as "non-aggregates" which are creating using `Aggregate.groupof(1)`

Reply:
If we did do that, I would want to make sure that we didn't have a bunch of conditionals everywhere checking for aggregation status. This might be outside the scope of GSoC, and need to be a later refactoring.

Reply:
Actually in #336 I have implemented such behaviour.
I checked if its a group of 1 with no `select`/`sort`/`reverse` parameters then we consider that as non aggregate and we don't print it in the aggregate details of status check.

Below is the sample status output

```
Overview:                                                                                                                                 

Total # of jobs: 50

label              ratio
-----------------  --------------------------------------------------
identity_computed  |████████████████████████████████████████| 100.00%

operation           number of eligible aggregates
----------------  -------------------------------
compute_identity                               50
compute_sum                                     1

Detailed View:

job_id                            operation             aggregation_status    labels
--------------------------------  --------------------  --------------------  -----------------
ee5ca9ab62e9dbb7b6abbaaac6443d49  compute_identity [U]  non-aggregate         identity_computed
                                  compute_sum [U]       aggregate - 50        identity_computed
03086200817396c6083c34ac025ec4d5  compute_identity [U]  non-aggregate         identity_computed
                                  compute_sum [U]       aggregate - 50        identity_computed

Detailed Aggregate View:

operation    jobs_in_aggregate                   length_of_aggregate  status
-----------  --------------------------------  ---------------------  --------
compute_sum  ee5ca9ab62e9dbb7b6abbaaac6443d49                     50  [U]
compute_sum  03086200817396c6083c34ac025ec4d5                     50  [U]
compute_sum  92f821919b4b0a2f15d0ef3f5d433550                     50  [U]
compute_sum  5d63db8dc4821a190f690fd66e4dd0be                     50  [U]
compute_sum  6acfe44c405aca35383ffb75561e8a7d                     50  [U]

```